### PR TITLE
Implementation of #137

### PR DESCRIPTION
**Pull request by ScoutAtlas**
#137

### CODE REVIEW

File: frontend/src/components/cards/miniCard.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 }
 
 .MiniCardImageContainer:hover {
-    border-color: var(--global--active--highlighting-color);
+    border-color: #ae56ff;
```
Comment (Waldleufer) [inline frontend/src/components/cards/miniCard.css:12]: Can't we have a variable here? would probably be better for readability and such ._.

File: frontend/src/components/mydashboard/MydashboardItemComponent.js [hunk 2]
```diff
@@ -82,6 +83,37 @@ function MydashboardItemComponent() {
...
+                    <p> {title} </p>
+                    <div className='dashboardMiniTitle'>
+                        <p>
+                            <i class="fa fa-line-chart" aria-hidden="true"/>
+                            {value} 
+                        </p>
+                    </div>
+                    
+                </div>
+            </Column>
```
Comment (Waldleufer) [inline frontend/src/components/mydashboard/MydashboardItemComponent.js:113]: Please swap all simple-flexbox components out for react-grid-system elements!

**Review by Waldleufer — COMMENTED** [review]
I would like to suggest the removal of the charts from the dashboard. 

And the switchout of the simple flexbox stuff.  it should all be react-grid-system Columns and Rows for compatibility sake.

I also found this: 
image
;) 

looking really snazzy otherwise! I like it!

File: frontend/src/components/mydashboard/MydashboardItemComponent.js (hunk 2 above)
Comment (Waldleufer) [reply frontend/src/components/mydashboard/MydashboardItemComponent.js]: I did it.

File: frontend/src/components/details/ColumnChartComponent.jsx [hunk 3]
```diff
@@ -63,15 +63,15 @@ const ColumnChartComponent = () => {
...
             responsive: [
                 {
                     breakpoint: 6400,
                     options: {
                         chart: {
-                            width: 500
+                            width: 600
                         },
                         legend: {
                             position: 'bottom'
```
Comment (MartinDuersch) [inline frontend/src/components/details/ColumnChartComponent.jsx:77]: somewhat more fitting than before

File: frontend/src/components/details/PieChartComponent.jsx [hunk 4]
```diff
@@ -45,7 +45,7 @@ const PieChartComponent = () => {
                 breakpoint: 6400,
                 options: {
                     chart: {
-                        heigt: 'auto'
+                        height: 'auto'
                     },
```
Comment (MartinDuersch) [inline frontend/src/components/details/PieChartComponent.jsx:49]: 🙃 🙃 🙃 🙃

File: frontend/src/components/header/HeaderComponent.js [hunk 5]
```diff
@@ -1,12 +1,12 @@
 import React, { useContext } from 'react';
 import { string } from 'prop-types';
-import { Row } from 'simple-flexbox';
+import { Container, Row } from 'react-grid-system';
 import { SidebarContext } from 'hooks/useSidebar';
```
Comment (MartinDuersch) [inline frontend/src/components/header/HeaderComponent.js:4]: grid system highly recommended

File: frontend/src/components/loading/LoadingComponent.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { createUseStyles, useTheme } from 'react-jss';
-import { Column } from 'simple-flexbox';
+import { Col } from 'react-grid-system';
```
Comment (MartinDuersch) [inline frontend/src/components/loading/LoadingComponent.js:5]: same here

File: frontend/src/components/login/LoginComponent.js [hunk 7]
```diff
@@ -13,13 +15,19 @@ import React from 'react';
 function LoginComponent() {
     const [, setState] = useContext(GlobalContext);
 
+    const dispatch = useDispatch();
+
```
Comment (MartinDuersch) [inline frontend/src/components/login/LoginComponent.js:19]: nice dispatching

File: frontend/src/components/productGrid/ProductGridComponent.js [hunk 8]
```diff
@@ -20,29 +20,66 @@ import { loadProjects } from 'store/actions/productAction';
...
+                    products = preProducts?.industrialApplication?.services;
+                    break;
+                default:
+                    break;
+            }
+            break;
+        default:
+            products = [];
+            break;
+    }
```
Comment (MartinDuersch) [inline frontend/src/components/productGrid/ProductGridComponent.js:79]: still a nice switch case :)

File: frontend/src/interface/BackendConnect.js [hunk 9]
```diff
@@ -19,8 +20,8 @@ export async function getSimaProProjects() {
             const items = data;
             result = items.data.Result.Data;
         });
-    //console.log('API call to get the list of Products');
-    //console.log(result);
+    console.log('API call to get the list of Products');
+    console.log(result);
```
Comment (MartinDuersch) [inline frontend/src/interface/BackendConnect.js:24]: never gets old

File: frontend/src/interface/BackendConnect.js [hunk 10]
```diff
@@ -31,6 +32,13 @@ export async function getSimaProProjects() {
  *   If the calculation is stored returns the results of calculation here.
  */
 export async function postCalculationRequest(projectId, callback) {
+    // Bypass using the SimaPro Data
+    const useDummyData = true;
+
+    if (useDummyData) {
+        return dummyPostCalculationRequest(projectId, callback);
+    }
+
```
Comment (MartinDuersch) [inline frontend/src/interface/BackendConnect.js:41]: nice if usage 🙃

File: frontend/src/interface/BackendConnect.js [hunk 11]
```diff
@@ -47,6 +55,10 @@ export async function postCalculationRequest(projectId, callback) {
         });
 }
 
+function dummyPostCalculationRequest(projectId, callback) {
+    dummyProcessBackendData(projectId, callback);
+}
+
```
Comment (MartinDuersch) [inline frontend/src/interface/BackendConnect.js:61]: nice function usage 🙃

File: frontend/src/interface/processBackendData.js [hunk 12]
```diff
@@ -72,3 +74,119 @@ export function processBackendData(data, callback) {
 
     callback();
 }
+
+/**
+ * Works like processBackendData but without SimaPro.
```
Comment (MartinDuersch) [inline frontend/src/interface/processBackendData.js:79]: hate to see this againt but we are out of choices i guess

File: frontend/src/resources/globalConstants/SimaProProductCategories.js [hunk 13]
```diff
@@ -0,0 +1,14 @@
...
+    generation: 'Generation',
+    transmission: 'Transmission',
+    industrialApplications: 'IndustrialApplication'
+};
+
+export const simaProTypes = {
+    product: 'Product',
+    solution: 'Solution',
+    service: 'Service'
+};
```
Comment (MartinDuersch) [inline frontend/src/resources/globalConstants/SimaProProductCategories.js:14]: constants, still good

File: frontend/src/routes/PrivateRoutes.js [hunk 14]
```diff
@@ -48,15 +48,23 @@ function PrivateRoutes() {
...
                         path={SLUGS.industrialApplications + '/:type'}
-                        component={ProductSolutionsServices}
+                        render={() => (
+                            <ProductSolutionsServicesComponent
+                                category={categories.industrialApplications}
+                            />
+                        )}
                     />
 
                     <Route
```
Comment (MartinDuersch) [inline frontend/src/routes/PrivateRoutes.js:70]: the changes never end with this pull request

File: frontend/src/store/index.js [hunk 15]
```diff
@@ -1,8 +1,12 @@
...
-    products: productReducer,
-  },
-})
\ No newline at end of file
+    reducer: {
+        industrialApplications: industrialApplicationsReducer,
+        generation: generationReducer,
+        transmission: transmissionReducer
+    }
+});
```
Comment (MartinDuersch) [inline frontend/src/store/index.js:12]: redux usage also good looking on this branch

**Review by MartinDuersch — APPROVED** [review]
Looks good, considering we are still at dummy data

**Review by manifau — COMMENTED** [review]
:)

**Pull request merged**